packages/fx-core/src/component/driver/aad/create.ts

```diff
@@ -90,7 +92,17 @@ export class CreateAadAppDriver implements StepDriver {
           )
         );
         // Create new AAD app if no client id exists
-        const aadApp = await aadAppClient.createAadApp(args.name);
+        // Throw error if unexpected signInAudience
+        if (
+          args.signInAudience &&
+          !(<any>Object).values(SignInAudience).includes(args.signInAudience)
```

Comment:
can we use Object.values(...) here? It will be readable.

Reply:
Resolved according to offline sync.